gemrb/core/GameScript/GameScript.cpp

```diff
@@ -2061,7 +2061,7 @@ bool GameScript::Update(bool *continuing, bool *done)
 	bool continueExecution = false;
 	if (continuing) continueExecution = *continuing;
 
-	RandomNumValue=RNG_SFMT::getInstance()->rand();
+	RandomNumValue=RAND_DEFAULT_MIN_MAX();
```

Comment:
Surely we can find a nicer name. I know macros don't support overloading by default and let's not add hacks, but how about `RAND_FULL()`, `RAND_ALL()` or something along those lines? 

And you can add spaces around the equality sign at the same time. :)

Reply:
Was also not sure about that, I'll take the `ALL` then.